lib/full-site-editing/template-parts.php

```diff
@@ -163,11 +163,11 @@ function set_unique_slug_on_create_template_part( $post_id ) {
 add_action( 'save_post_wp_template_part', 'set_unique_slug_on_create_template_part' );
 
 /**
- * Returns a filtered list of allowed area types for template parts.
+ * Returns a filtered list of allowed area values for template parts.
  *
  * @return array The supported template part area types.
  */
-function gutenberg_get_allowed_template_part_area_types() {
+function gutenberg_get_allowed_template_part_area_values() {
```

Comment:
Is `gutenberg_get_allowed_template_part_areas` a good alternative or not?

Reply:
yeah i think that works for this function as well.  updated.